### PR TITLE
[xml/de] Created german version of xml and brainfuck

### PR DESCRIPTION
**Pull request by igeligel**
I created xml-de.html.markdown and fixed one missspell in xml.html.markdown.

### CODE REVIEW

File: de-de/brainfuck-de.html.markdown [hunk 1]
```diff
@@ -0,0 +1,96 @@
+---
+language: brainfuck
+filename: learnbrainfuck-fr.bf
```
Comment (vendethiel) [inline de-de/brainfuck-de.html.markdown]: -fr?

Comment (igeligel) [inline de-de/brainfuck-de.html.markdown]: Sry copy&paste error. Fixed it :)

File: de-de/brainfuck-de.html.markdown [hunk 2]
```diff
@@ -0,0 +1,95 @@
+---
+language: brainfuck
```
Comment (vendethiel) [inline de-de/brainfuck-de.html.markdown]: please add a `filename`

**Comment by vendethiel** [comment]
can you please fix the merge conflicts? Thanks, and sorry for the delay!

**Comment by vendethiel** [comment]
Closing in the meantime :). Ping me if you find some time to fix those, and I'll gladly reopen.

**Pull request closed without merging**